addons/website_event_stand/data/event_stand_demo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo><data>
+
+    <record id="event.event_0" model="event.event">
+        <field name="website_menu" eval="True"/>
+        <field name="website_stand" eval="True"/>
+    </record>
+
+    <record id="event.event_2" model="event.event">
+        <field name="exhibition_map" type="base64" file="website_event_stand/static/src/img/exhibition-map.gif"/>
```

Comment:
Bioutiful map :) It would be even better if you could use a SVG file instead of a GIF (to gain some bytes :) )